### PR TITLE
Update audio-conferencing-subscription-dial-out.md

### PR DESCRIPTION
**Pull request by ruizoscar**
Making updates to the document to reflect a recent change in product behavior.

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Teams/audio-conferencing-subscription-dial-out.md [hunk 1]
```diff
@@ -28,10 +28,14 @@ description: Dial-Out and Call Me At minutes benefit. As of December 1, 2019 eac
...
+Each Audio Conferencing subscription provides 60 minutes per user per month that can be used to dial out to non-premium numbers in ANY of the Zone A countries as described in this document. Your tenant dial-out minute pool size is based on *purchased* licenses. This benefit is applicable to Audio Conferencing *monthly subscription* licenses and does not extend to Audio Conferencing pay-per-minute licenses. 
 
 > [!NOTE]
-> The end of the complimentary dial-out period will not take place on November 30, 2019, for countries where the Audio Conferencing subscription is available, but we do not currently provide the ability to set up Communications Credits. These specific countries are Russia, South Korea, and Taiwan.
+> As of October 22, 2020, the dial-out minute pool size of our tenant is based on the number of *purchased* Audio Conferencing subscription licenses. Before, the dial-out minute pool size was based on the number of licenses that were *assigned* to users.
```
Comment (JohanFreelancer9) [inline Teams/audio-conferencing-subscription-dial-out.md:34]:
```suggestion
> As of October 22, 2020, the dial-out minute pool size of your tenant is based on the number of *purchased* Audio Conferencing subscription licenses. Before, the dial-out minute pool size was based on the number of licenses that were *assigned* to users.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Comment by scanum** [comment]
@tonysmit PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Pull request merged**